test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -2457,9 +2457,15 @@ public void ensureEstimatedStats() {
                 final CircuitBreakerService breakerService = getInstanceFromNode(CircuitBreakerService.class, nodeAndClient.node);
                 CircuitBreaker fdBreaker = breakerService.getBreaker(CircuitBreaker.FIELDDATA);
                 assertThat("Fielddata breaker not reset to 0 on node: " + name, fdBreaker.getUsed(), equalTo(0L));
-                CircuitBreaker acctBreaker = breakerService.getBreaker(CircuitBreaker.ACCOUNTING);
-                assertThat("Accounting breaker not reset to 0 on node: " + name + ", are there still Lucene indices around?",
-                        acctBreaker.getUsed(), equalTo(0L));
+                try {
+                    assertBusy(() -> {
+                        CircuitBreaker acctBreaker = breakerService.getBreaker(CircuitBreaker.ACCOUNTING);
+                        assertThat("Accounting breaker not reset to 0 on node: " + name + ", are there still Lucene indices around?",
+                            acctBreaker.getUsed(), equalTo(0L));
+                    });
+                } catch (Exception e) {
+                    fail("Exception during check for accounting breaker reset to 0: " + e);
```

Comment:
This will throw an AssertionError with a string representation of `e` in the message. I'd rather like `e` to be the cause of the raised exception.

Reply:
Sounds good, I changed this.